dumb.rb

```diff
@@ -6,3 +6,6 @@ def bad_spacing
 
 dumb = Dumb.new
 p dumb.bad_spacing
+
+
+  message = "this line has extra space everywhere"  
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)